core/templates/components/skill-selector/select-skill-modal.controller.spec.ts

```diff
@@ -58,7 +59,7 @@ describe('Select Skill Modal Controller', function() {
     expect($scope.countOfSkillsToPrioritize).toBe(skillsInSameTopicCount);
   });
 
-  it('should save skill with its id', function() {
+  it('should save skill with its id when saving modal', function() {
```

Comment:
I don't "save" a modal. I think you mean when a save button is clicked?

Reply:
I meant when modal is being closed.